src/commands/Tools/avatar.ts

```diff
@@ -1,10 +1,14 @@
 import { DbSet } from '@lib/structures/DbSet';
 import { SkyraCommand } from '@lib/structures/SkyraCommand';
-import { MessageEmbed } from 'discord.js';
+import { MessageEmbed, ImageExt, ImageSize } from 'discord.js';
 import { CommandStore, KlasaMessage, KlasaUser } from 'klasa';
 
+const VALID_SIZES = [32, 64, 128, 256, 512, 1024, 2048];
+
 export default class extends SkyraCommand {
 
+	private kExtensionRegex = /.?(png|jpg|webp|gif)/i;
```

Comment:
Escape the dot and allow jpeg
```suggestion
	private kExtensionRegex = /\.?(png|jpe?g|webp|gif)/i;
```